Tests/CommandsTests/BuildCommandTests.swift

```diff
@@ -1011,7 +1162,14 @@ struct BuildCommandTestCases {
     @Test(
         .bug("https://github.com/swiftlang/swift-package-manager/issues/8659", "SWIFT_EXEC override is not working"),
         .SWBINTTODO("Test fails because the dummy-swiftc used in the test isn't accepted by swift-build. This needs to be investigated"),
-        arguments: getBuildData(for: SupportedBuildSystemOnPlatform),
+        .tags(
+            .Feature.EnvironmentVariables.SWIFT_EXEC,
```

Comment:
Same question here, I'm not 100% convinced tags for individual env vars makes sense